### PR TITLE
Chore/migrate to GitHub actions

### PR DESCRIPTION
**Pull request by snim2**
Also deprecated PHP versions 7.0 and 7.1.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   },
   "config": {
     "platform": {
-        "php": "7.0"
+        "php": "7.2"
```
Comment (RobjS) [inline composer.json]: This could be 7.4 now, we don't run anything below that.

File: .github/workflows/continuous-integration.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+on:
+  pull_request:
+  push:
+
+jobs:
+  lint-and-test:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        php-versions: ['7.2', '7.3', '7.4']
```
Comment (RobjS) [inline .github/workflows/continuous-integration.yml]: This could probably just be 7.4, we don't support anything below that.

**Review by RobjS — COMMENTED** [review]
Just one point on making it 7.4 only.

File: .github/workflows/continuous-integration.yml (hunk 2 above)
Comment (snim2) [reply .github/workflows/continuous-integration.yml]: OK, pushed a commit for both of those files

**Review by RobjS — APPROVED** [review]

**Pull request merged**